common/src/main/java/com/wynntils/features/user/DummyOverlayFeature.java

```diff
@@ -8,47 +8,106 @@
 import com.mojang.blaze3d.vertex.PoseStack;
 import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.features.overlays.BasicOverlay;
+import com.wynntils.core.features.overlays.Coordinate;
 import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayManager;
 import com.wynntils.core.features.overlays.OverlayPosition;
 import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.overlaySizes.GuiScaleRespectingOverlaySize;
 import com.wynntils.mc.event.RenderEvent;
 import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.utils.McUtils;
 import com.wynntils.utils.objects.CustomColor;
+import com.wynntils.utils.objects.Pair;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class DummyOverlayFeature extends UserFeature {
 
     @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
     private final Overlay ComplexRedOverlay = new DummyRedComplexOverlay();
 
     @OverlayInfo(renderType = RenderEvent.ElementType.GUI, renderAt = OverlayInfo.RenderState.Pre)
-    private final Overlay BasicBlueOverlay =
-            new BasicOverlay(new OverlayPosition(50, 50), 75, 75, DummyOverlayFeature::renderBasicBlueBox);
+    private final Overlay BasicBlueOverlay = new BasicOverlay(
+            new OverlayPosition(
+                    15,
+                    15,
+                    OverlayPosition.VerticalAlignment.Top,
+                    OverlayPosition.HorizontalAlignment.Left,
+                    OverlayPosition.AnchorNinth.TopLeft),
```

Comment:
minor detail for now, but this should be defined in its own field in the feature class so it can be collected as a config option

Reply:
actually maybe it's a better idea to just register them from `OverlayManager`, since we always know exactly what we're dealing with and what needs to be registered - no need to deal with annotation handling

this would require a special method in `ConfigManager` but that's fine

Reply:
What do you mean? Do you want to have OverlayPosition fields declared in the feauture, with @Config annotation and having those passed to the Overlay constructors?

Reply:
Also, why can't we just look for Overlays and get the position field for them and load those?

Reply:
> What do you mean? Do you want to have OverlayPosition fields declared in the feauture, with @config annotation and having those passed to the Overlay constructors?

that was my original thought yeah, but then i realized there's better ways of accomplishing what we need


> Also, why can't we just look for Overlays and get the position field for them and load those?

not sure what you mean by "look for", since we already handle/register each overlay in `OverlayManager` - we'd just need a bit of extra code to extract the position field (which is easy since we know exactly what it is, no annotation searching required) and then register it via `ConfigManager`

Reply:
Yeah, I agree. So, if we understand each other correctly, no change needed atm? (and we will get back to this in the overlay config saving pr)

Reply:
yeah, no need to implement it with this PR